dulwich/porcelain.py

```diff
@@ -1598,7 +1603,7 @@ def active_branch(repo):
         active_ref = r.refs.follow(b"HEAD")[0][1]
         if not active_ref.startswith(LOCAL_BRANCH_PREFIX):
             raise ValueError(active_ref)
-        return active_ref[len(LOCAL_BRANCH_PREFIX) :]
```

Comment:
unnecessary whitespace change

Reply:
This one was to fix `PEP 8: E203 whitespace before ':'` PEP8 warning.  Just checking that you want the space before the ':'.

Reply:
Not particularly (dulwich is meant to follow PEP8), but it's unrelated to the other changes.

Reply:
[daaf927](https://github.com/jelmer/dulwich/pull/1148/commits/daaf92711c725e989d670761a961ffd2ab3d59e5)